WebUI/lims/views.py

```diff
@@ -38,28 +38,59 @@ def index(request, pageNum = None):
     for ndx in range(0, len(rows)):
         rowsList.append(list(rows[ndx]))
         rowsList[ndx][2] = rowsList[ndx][2].strftime(DATEFORMAT)
+    if not pageNum:
+        return redirect('/1')
     pageNumberNext, newRow = nextPage_newRow(pageNum,rowsList)
-    newPathPart = '/'
+    newPathPart = ''
```

Comment:
Instead of redirecting and depending upon the browser to request a new URL, would it be possible to just assume you are on the first page behind the scenes?